src/main/java/com/scaleset/cfbuilder/lambda/Function.java

```diff
@@ -0,0 +1,58 @@
+package com.scaleset.cfbuilder.lambda;
+
+import com.scaleset.cfbuilder.annotations.Type;
+import com.scaleset.cfbuilder.core.Taggable;
+
+/**
+ required properties are:
+ code
+ handler
+ role
+ runtime
+ */
+@Type("AWS::Lambda::Function")
+public interface Function extends Taggable {
+
+    /**
+     required
+     */
+    Function code(Object value);
```

Comment:
Can this contain anything but a Code object?
Or would this work with code(Code value)?

In general, we should probably work on making the builder more type-safe and more compliant with the CloudFormation spec (e.g. required properties, conditionals etc.).
Since this isn't really an issue that just affects this PR, but the whole repo, I made new issues for this:
- #26
- #27